<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeypadWidget</class>
 <widget class="QWidget" name="KeypadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <spacer name="rightSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btn_entrar">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>64</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #51B323;
border: none;
color: white;</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="line_numeros_ingresados">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>51</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="leftSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QWidget" name="numpad" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;

</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;
</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_0">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>128</width>
          <height>64</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: white;
border: none;
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_numeros_ingresados</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_0</tabstop>
  <tabstop>btn_entrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
